0020-py-version-support.md

```diff
@@ -0,0 +1,258 @@
+- Feature Name: Python Version Support
+- Start Date: 2020-03-24
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This SEP is being created to add a policy around Python versions Salt will support and
+detail how we will support Python versions going forward. This policy change is to align Salt Releases
+with Python's End of Life schedule.
+
+# Motivation
+[motivation]: #motivation
+
+Documentation and Communication
+-------------------------------
+One of the motivating factors around this policy change is to clarify our Python version support
+for each Salt release. If it is clearly defined, then packagers, community members, and Salt users will
+know when a Python version will be dropped for a given Salt release. In the past we had never clearly
+defined how our Python version support lined up with Python's End of Life schedule. We previously made a
+best effort to support the Python versions that lined up with the operating systems Python versions, but never clearly
+defined this and did not always follow this in every case. For example, when we dropped Python 2.6 support
+we still supported Red Hat 6, which had Python 2.6 installed by default. To continue supporting Red Hat 6 while also
+removing support for the End of Life Python 2.6 version, we included Python 2.7 in our packages
+for that operating system.
+
+Including Python in Packages
+----------------------------
+Another motivating factor to make this policy change is our shift to including Python in our
+packages. For the Sodium release we will be using salt-bin (https://github.com/saltstack/salt-bin) to
```

Comment:
Salt-bin is now pop-build https://gitlab.com/saltstack/pop/pop-build/

Reply:
```suggestion
packages. For the Sodium release we will be using pop-build (https://gitlab.com/saltstack/pop/pop-build/) to
```